build.gradle

```diff
@@ -1,36 +1,44 @@
 
-plugins {
-    id 'net.researchgate.release' version '2.3.4'
+buildscript {
+    repositories {
+        maven { url 'http://repo.spring.io/plugins-release' }
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
+    dependencies {
+        classpath 'org.springframework.build.gradle:propdeps-plugin:0.0.7'
+        classpath 'gradle.plugin.org.jetbrains:gradle-intellij-plugin:0.0.29'
+    }
 }
 
-apply plugin: 'base'
-
-task loadIdea(type:Exec) {
-    commandLine './fetchIdea.sh'
+configure(allprojects) {
```

Comment:
do these have to be applied at the "allprojects" level? or can be at the "subprojects" level?